internal/executors/http/basic_authorization_header.go

```diff
@@ -18,6 +19,7 @@ func NewBasicAuthorizationHeader(u string, p string) AuthorizationHeaderGenerato
 }
 
 func (h *basicAuthorizationHeader) Generate() (string, error) {
+	log.Println("Basic Authorization Header: generating auth header...")
```

Comment:
While the authorization headers are part of the whole http request flow, I thought it was more important to prefix the logs with _what_ originated them, as in, what class, rather than just their semantic origin. So in the case with logs from different auth header classes, my opinion is that it's not needed to prefix them with `HTTP Client`.